nautobot/circuits/filters.py

```diff
@@ -141,45 +141,45 @@ class CircuitFilterSet(NautobotFilterSet, StatusModelFilterSetMixin, TenancyMode
         label="Provider (slug)",
     )
     provider_network_id = django_filters.ModelMultipleChoiceFilter(
-        field_name="terminations__provider_network",
+        field_name="circuit_terminations__provider_network",
         queryset=ProviderNetwork.objects.all(),
         label="Provider Network (ID)",
     )
-    type_id = django_filters.ModelMultipleChoiceFilter(
+    circuit_type_id = django_filters.ModelMultipleChoiceFilter(
         queryset=CircuitType.objects.all(),
         label="Circuit type (ID)",
     )
-    type = django_filters.ModelMultipleChoiceFilter(
-        field_name="type__slug",
+    circuit_type = django_filters.ModelMultipleChoiceFilter(
+        field_name="circuit_type__slug",
         queryset=CircuitType.objects.all(),
         to_field_name="slug",
         label="Circuit type (slug)",
     )
```

Comment:
Can/should we go ahead and collapse these two to a `NaturalKeyOrPKMultipleChoiceFilter`?

Reply:
Would this be out of scope since this [story](https://github.com/nautobot/nautobot/issues/2808) should be covering this change?

Reply:
That's fine, sure. Probably good to keep this more narrowly scoped.